### PR TITLE
[docs]: Add environmental variables in README

### PR DESCRIPTION
**Pull request by shikharish**
Concerning issue #120

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -83,7 +83,23 @@ Docker and docker compose are the only required dependencies. You can either ins
 <p align="right">(<a href="#top">back to top</a>)</p>
 
 #### Environment Variables
-
+Environment variables can be set using a `.env` file(use `.env.template` file for reference). The following variables are used:
+
+- `DEV`: When set to `true`, stack-trace is shown with error messages.(Default: `false`)
```
Comment (harshkhandeparkar) [inline README.md]:
```suggestion
- `DEV`: When set to `true`, Mediawiki PHP stack-trace is shown with error messages. (Default: `false`)
```

File: README.md [hunk 2]
```diff
@@ -83,7 +83,23 @@ Docker and docker compose are the only required dependencies. You can either ins
...
+- `MYSQL_PASSWORD`: A secret password for the MySQL database.
```
Comment (harshkhandeparkar) [inline README.md:89]:
```suggestion
- `MYSQL_PASSWORD`: A randomly generated secret password for the MySQL database.
```

File: README.md [hunk 3]
```diff
@@ -83,7 +83,23 @@ Docker and docker compose are the only required dependencies. You can either ins
...
+- `SERVER_PORT`: Port on which the server is exposed to the host(Default: `8080`)
```
Comment (harshkhandeparkar) [inline README.md]:
```suggestion
- `SERVER_PORT`: Port on which the wiki server is exposed to the host. (Default: `8080`)
```

File: README.md [hunk 4]
```diff
@@ -83,7 +83,23 @@ Docker and docker compose are the only required dependencies. You can either ins
...
+- `SERVER_NAME`: Base URL of the wiki.
+- `MAILGUN_PASSWORD`: Mailgun SMTP password for sending official mails from the wiki.
+- `GOOGLE_ANALYTICS_ACCOUNT`: Google Analytics Account (GA-XXXXXX or UA-XXXXXX)
```
Comment (harshkhandeparkar) [inline README.md]: Let's not add this for now since the GA jobs are broken.

File: README.md [hunk 5]
```diff
@@ -83,7 +83,23 @@ Docker and docker compose are the only required dependencies. You can either ins
...
+- `GOOGLE_ANALYTICS_SERVICE_KEY`: Google Analytics Service Key (Only used in production to update statistics)
+- `WG_SECRET_KEY`: Secret key used for encryption by mediawiki. Make it a long, random, secret string([Reference](https://www.mediawiki.org/wiki/Special:MyLanguage/Manual:$wgSecretKey)).
+- Dropbox related variables(used for storing backups): 
+	- `DROPBOX_APP_KEY`:  Dropbox app key(can be found at [Dropbox App Console](https://www.dropbox.com/developers/apps)).
+	- `DROPBOX_APP_SECRET`:  Dropbox app secret(can be found at [Dropbox App Console](https://www.dropbox.com/developers/apps)).
+	- `DROPBOX_ACCESS_TOKEN`: Dropbox API access token(generated using `/scripts/get_dropbox_tokens.py`)
+	- `DROPBOX_REFRESH_TOKEN`: Dropbox API refresh token(generated using `/scripts/get_dropbox_tokens.py`) used to refresh the access token.
+- `SLACK_WEBHOOK_URL`: URL to the Slack webhook used to send updates.
+- `BATMAN_BOT_PASSWORD`: A generated password of the Batman bot user account on the wiki.
```
Comment (harshkhandeparkar) [inline README.md]: Can you link to a mediawiki docs article that explains how to generate a bot password?

**Review by harshkhandeparkar — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (shikharish) [reply README.md:89]: is this randomly generated? In local setup we can set it what we want.

File: README.md (hunk 5 above)
Comment (shikharish) [reply README.md]: Done.

File: README.md (hunk 2 above)
Comment (harshkhandeparkar) [reply README.md:89]: I meant to say that the user should randomly generate a strong password. This is fine too.

**Review by harshkhandeparkar — APPROVED** [review]

**Review by rajivharlalka — APPROVED** [review]

**Pull request merged**